nih-sparc/sparc-app
# File detail page #4m8bwn

# Description

The purpose of this PR is to add a file detail page. In addition, the Biolucida viewer will now be hidden if there is no viewer available for that file.

## Type of change


# How Has This Been Tested?

- Go to Find Data
- Open a dataset
- Go to the Files tab
- Click on a file's name that is not a microsoft office file
- You should be taken to a file detail page displaying that file's data
- Click on "Download" (you will need the sparc-api running locally.) If you don't have it running locally, you can verify that the call is made in the network tab.

# Checklist:
